src/components/PeopleTable/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,55 @@
+import { useCallback, useMemo } from 'react';
+import { Person } from '../../../types';
+import { PersonLink } from './PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  personToRender: Person;
+  slug?: string;
+  people: Person[];
+};
+
+export const PeopleRow: React.FC<Props> = ({
+  personToRender,
+  slug,
```

Comment:
You don't need to pass slug as a property, apply useSearchParams directly in the component

Reply:
nice idea, but we haven't come to this topic yet)